src/config/dynamic-theme-fixes.config

```diff
@@ -45,6 +45,18 @@ div[class^='scrollableList'] {
 
 ================================
 
+*.readthedocs.io
```

Comment:
In this case you don't need `*`, I might need to change the other ones that has this, as it currently has zero effect.